src/main/resources/jenkins/branch/Messages.properties

```diff
@@ -28,6 +28,9 @@ DescriptionColumn.displayName=Project description
 ItemColumn.DisplayName=Name
 NamedExceptionsBranchPropertyStrategy.DisplayName=Named branches get different properties
 NoTriggerBranchProperty.suppress_automatic_scm_triggering=Suppress automatic SCM triggering
+NoTriggerBranchProperty.strategy.all=Suppress all
+NoTriggerBranchProperty.strategy.indexing=Suppress only triggered by indexing
+NoTriggerBranchProperty.strategy.events=Suppress only triggered by events
```

Comment:
```suggestion
NoTriggerBranchProperty.strategy.all=Suppress all, runs will not be trigger by indexing or events
NoTriggerBranchProperty.strategy.indexing=Suppress runs triggered by indexing
NoTriggerBranchProperty.strategy.events=Suppress runs triggered by webhook events
```

Reply:
I added the `webhook` word to the `NoTriggerBranchProperty.strategy.all` property value too, to keep docs consistent.

```
NoTriggerBranchProperty.strategy.all=Suppress all, runs will not be trigger by indexing or webhook events
NoTriggerBranchProperty.strategy.indexing=Suppress runs triggered by indexing
NoTriggerBranchProperty.strategy.events=Suppress runs triggered by webhook events
```